Fix build when using a 32-bit chroot with a x86_64 kernel
Native tools would be built with -m64
--- configure.orig	2011-11-11 14:48:12.944666802 -0500
+++ configure	2011-11-11 14:48:55.108875882 -0500
@@ -168,7 +168,7 @@
 #-------------------------------------------------------------------------------
 
 # need that throughout the script
-UNAME_MACHINE=`(uname -m) 2>/dev/null` || UNAME_MACHINE=unknown
+UNAME_MACHINE=`(gcc -dumpmachine|cut -d- -f1) 2>/dev/null` || UNAME_MACHINE=unknown
 UNAME_RELEASE=`(uname -r) 2>/dev/null` || UNAME_RELEASE=unknown
 UNAME_SYSTEM=`(uname -s) 2>/dev/null`  || UNAME_SYSTEM=unknown
 UNAME_VERSION=`(uname -v) 2>/dev/null` || UNAME_VERSION=unknown
